docs/astro/sql/run_raw_sql.rst

```diff
@@ -0,0 +1,20 @@
+====================
+run_raw_sql Operator
+====================
+
```

Comment:
@kaxil @tatiana - Should we follow a pattern/standard here?

I'm following in load_file and cleanup operator
1. When should we use this operator
2. Covering various cases of operator

or if you guys have other suggestions, I think it's essential to have one format and stick to it. WDYT?

Reply:
I think that is a great idea, @utkarsharma2 !

@pankajastro in this particular operator, I think this gives an overall idea of why we have this operator - but you can rephrase it more cleanly:

We encourage users to use one of the specific decorators (load_file for loading, transform for selecting statements / outputting the content to a table), but there are scenarios when those operators may not meet the users' needs (e.g. change only makes sense for statements which output a table).

For this reason, we also offer the`run_raw_sql` (decorator/task?). It allows users to declare any SQL statement using the Astro SDK templating available in `transform` It also empowers users to define the task output (e.g. a list with the row results, etc) by using the `handler` argument.

To be confirmed: users can pass the `sql` argument, and if it references a file which ends in `.sql`, it will run the content of the SQL file.